core_backend/app/question_answer/models.py

```diff
@@ -131,6 +134,11 @@ async def save_query_response_to_db(
         llm_response=response.model_dump()["llm_response"],
         response_datetime_utc=datetime.utcnow(),
     )
+
```

Comment:
You don't need to do that since you gave a default value to `tts_file` in the pydantic model.